sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/AggUtils.scala

```diff
@@ -121,7 +147,7 @@ object AggUtils {
     // If we have session window expression in aggregation, we add MergingSessionExec to
     // merge sessions with calculating aggregation values.
     val interExec: SparkPlan = mayAppendMergingSessionExec(groupingExpressions,
-      aggregateExpressions, partialAggregate)
+      aggregateExpressions, partialAggregate, isStreaming = false)
```

Comment:
nit: how about using default value in `mayAppendMergingSessionExec` and let here unchange?